Birthdays/Birthdays/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Birthdays
+{
+    class Program
+    {
+        public static Dictionary<String, List<string>> dictionary = new Dictionary<string, List<string>>();
+        static void Main()
+        {
+            var month = new int[12];
+
+            try
+            {
+                using(StreamReader sr = new StreamReader("birthday.txt"))
+                {
+                    string line;
+                    while((line = sr.ReadLine()) != null)
+                    {
+                        ParseLineString(line);
+                    }
+                }
+            }
+            catch { }
+
+            while (true)
+            {
+                Console.WriteLine("Введите имя");
+                var name = Console.ReadLine();
+                if (name == "") break;
+                if(!dictionary.ContainsKey(name))
+                {
+                    Console.WriteLine("Данного имени нет в списке");
+                    continue;
+                }
+                month = CountBirthdays(name);
+                WriteDataInConsole(month);
+                Console.ReadLine();
+            }
+        }
+
+        private static void WriteDataInConsole(int[] month)
+        {
+            Console.WriteLine();
+            for (int i = 0; i < month.Length; i++)
+                Console.Write($"{i + 1}         ");
+            Console.WriteLine();
+            for(int i = 0; i < month.Length; i++)
+                Console.Write($"{month[i]}         ");
```

Comment:
Вывод красивой таблицы по 6 символов на колонку (см. комментарий к строке 47):
```suggestion
                Console.Write($"{month[i]}, 6");
```